Prose/ProseLib/Sources/ConversationFeature/Chat/ChatReducer.swift

```diff
@@ -26,19 +26,30 @@ struct ChatState: Equatable {
 struct MessageReactionPickerState: Equatable {
   let messageId: Message.ID
   var pickerState: ReactionPickerState
-  let origin: CGPoint
+  let origin: CGRect
 }
 
 // MARK: - Actions
 
-public struct MessageMenuHandlerPayload: Equatable, Decodable {
-  struct Point: Equatable, Decodable {
-    let x, y: Double
-    var cgPoint: CGPoint { CGPoint(x: self.x, y: self.y) }
-  }
+struct CoreViewsPoint: Equatable, Decodable {
+  let x, y: Double
+  var cgPoint: CGPoint { CGPoint(x: self.x, y: self.y) }
+}
 
+struct CoreViewsFrame: Equatable, Decodable {
+  let x, y, width, height: Double
+  var cgRect: CGRect { CGRect(x: self.x, y: self.y, width: self.width, height: self.height) }
+}
+
+struct CoreViewsOrigin: Equatable, Decodable {
```

Comment:
`Origin` is a bit ambiguous here. Maybe we should follow suit and also call it `EventOrigin` like in the core views repo?

Reply:
Ok, it's just that I thought we wanted to create CoreViews- mapping structs. `EventOrigin` is quite the opposite, it reads very general. What about `CoreViewsEventOrigin`?

Reply:
Yes, that's what I meant.